app/view/show.py

```diff
@@ -17,9 +17,20 @@
 from collections import defaultdict, OrderedDict
 from jinja2.utils import escape
 
+def get_bug_project(databases):
+    bug_project_mapping = {
+        1: ['core', 'extra', 'testing'],
```

Comment:
I wonder if these number could use a constant declaration somewhere (could they change in the future?)